grpc_unary_attestation/src/server.rs

```diff
@@ -68,7 +71,7 @@ impl SessionTracker {
         match self.known_sessions.pop(&session_id) {
             None => match AttestationBehavior::create_self_attestation(&self.tee_certificate) {
                 Ok(behavior) => Ok(SessionState::HandshakeInProgress(Box::new(
-                    ServerHandshaker::new(behavior, self.additional_info.clone()),
+                    ServerHandshaker::new(behavior, Arc::new(self.additional_info.clone())),
```

Comment:
Is it not possible to move the Arc even higher up so that each session tracker also stores a reference to the same fixed additional info to avoid cloning the vector here?

Reply:
Possible to move it up higher, but there will only ever be on `SessionTracker`. That's why it was called `SessionsTracker` (plural) in my initial PR. 

What do you think would be a good way to make that clearer? Could add a comment above it

Reply:
true, updated it, thx